Sources/Client/Client.h

```diff
@@ -434,6 +434,8 @@ namespace spades {
 
 			void NetLog(const char* format, ...);
 
+			static Client* globalInstance;
```

Comment:
Even if global instance of client was needed. You do not need to expose the global instance of the client via a header file.